doc/sphinx/compiling.rst

```diff
@@ -39,6 +40,23 @@ Linux
 
       cython python-dev python-numpy python-numpy-dev python-setuptools
 
+* Building the python3 module requirs the following libraries::
+    
+      python3 python3-setuptools cython3 python3-numpy
+
+* For Fedora (version 22, or higher) users, the following packages should
+  be installed via the package manager::
+
+      g++ python scons sundials
+
+  If your Fedora version is lower than 22, `sundials` package is not available,
+  and you should build Sundials from sources, and two packages, `blas-devel`
+  and `lapack-devel`, as well.
```

Comment:
Actually, in the current development version, if you're working with a git checkout of the code and Sundials is not available, it will be automatically downloaded and compiled as part of the Cantera build process.

Also, I assume that this should say that the `blas-devel` and `lapack-devel` packages should be installed, not that the user should build BLAS and LAPACK from source.